WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderCommentListActivity.java

```diff
@@ -57,6 +60,13 @@
     private static final String KEY_REPLY_TO_COMMENT_ID = "reply_to_comment_id";
     private static final String KEY_HAS_UPDATED_COMMENTS = "has_updated_comments";
 
+    public enum DirectOperation {
```

Comment:
Can we add a comment to this enum describing its purpose? We use it in several places and it would be good to make its purpose more clear.